compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDmaLoopSubsumption.cpp

```diff
@@ -349,22 +386,29 @@ class SubsumeLoopIntoDMA
     //   be used for DMA access patterns.
     size_t sourceMaxNbDims{0};
     size_t targetMaxNbDims{0};
+    size_t shimNbIntraDims = (size_t)deviceModel.getDmaProp<uint8_t>(
```

Comment:
```suggestion
    uint8_t shimNbIntraDims = deviceModel.getDmaProp<uint8_t>(
```
Doesn't this work? Seems like then you add a uint8_t to a size_t, which the compiler shouldn't warn about ever as there's no risk

Reply:
That works as well